src/main/java/uk/gov/companieshouse/web/accounts/service/smallfull/impl/StocksServiceImpl.java

```diff
@@ -66,4 +73,29 @@ private StocksApi getStocksApi(String transactionId, String companyAccountsId) t
             throw new ServiceException(INVALID_URI_MESSAGE, e);
         }
     }
+    
+    @Override
+    public List<ValidationError> deleteStocks(String transactionId, String companyAccountsId) throws ServiceException {
+        ApiClient apiClient = apiClientService.getApiClient();
+
+        String uri = STOCKS_URI.expand(transactionId, companyAccountsId).toString();
+
+        try {
+            apiClient.smallFull().stocks().delete(uri).execute();
+        } catch (URIValidationException e) {
+
+            throw new ServiceException(INVALID_URI_MESSAGE, e);
+        } catch (ApiErrorResponseException e) {
+            if (e.getStatusCode() == HttpStatus.BAD_REQUEST.value()) {
```

Comment:
It shouldn't be possible to have a bad request when deleting a resource. It should be sufficient to keep this as a void method which can throw a `ServiceException`.

Reply:
Method is now void